### PR TITLE
REF: include row positioner templates for new typhos

### PR DESCRIPTION
**Pull request by klauer**
## Description
Switch BeckhoffAxis and TwinCATStatePositioner to the new row positioner widget in typhos

## Motivation and Context
https://github.com/pcdshub/typhos/pull/563

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by klauer** [comment]
I think we have some other templates that need adjusting here as well.

**Comment by ZLLentz** [comment]
I commented on the other PR before seeing this one- I'm happy to help normalize this here as needed.

**Comment by klauer** [comment]
Yeah, I think now that we have the other merged it's time to get this going. Any/all contributions here would be appreciated!

**Comment by ZLLentz** [comment]
I'm pushing a commit that includes the other needed template updates.
Then, I'm going to try to figure out https://github.com/pcdshub/typhos/issues/585 because it's somewhat a blocker.
I want to/need to put this PR and the typhos update into dev_conda at the same time, or else the screens will break. So, both need to be at some minimum threshold.

**Review by ZLLentz — APPROVED** [review]
I think we should merge this and update dev today

File: docs/source/upcoming_release_notes/1159-row_positioner.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- klauer
```
Comment (klauer) [inline docs/source/upcoming_release_notes/1159-row_positioner.rst:31]:
```suggestion
- klauer
- zllentz
```

**Pull request merged**